openapi.yml

```diff
@@ -730,6 +723,102 @@ components:
         - fiscal_code
         - given_name
         - role
+    UserDelegationRequest:
+      type: object
+      properties:
+        organization:
+          $ref: "#/components/schemas/Organization"
+        id:
+          type: integer
+        requester:
+          $ref: "#/components/schemas/OrganizationDelegate"
+        status:
+          $ref: '#/components/schemas/RequestStatus'
+        type:
+          type: string
+          x-extensible-enum:
+            - "USER_DELEGATION"
+      required:
+        - id
+        - organization
+        - requester
+        - status
+        - type
+      example:
+        {
+          id: 12345,
+          organization: {
+            "fiscal_code": "86000470830",
+            "ipa_code": "c_e043",
+            "legal_representative": {
+              "family_name": "Spano'",
+              "given_name": "Ignazio Alfonso",
+              "fiscal_code": "BCDFGH12A21Z123D",
+              "phone_number": "5550000000",
+              "email": "indirizzo00@email.pec.it",
+              "role": "ORG_MANAGER"
```

Comment:
About the `role` properties for this and the other cases, they are inferred by the client request because the only consistent role for the legal representative is `ORG_MANAGER` and the only consistent role for the requester is `ORG_DELEGATE`.
Indeed, the `role` properties inside a request object are redundant. They are used in the example objects because they are required for both the `OrganizationDelegate` and the `LegalRepresentative` schemas which are used inside the `Request` schema. Maybe I'd better change to the schemas in order to remove this redundancy?

Reply:
a question: is the role here the same of [userRole](https://github.com/teamdigitale/io-onboarding-pa-api/blob/master/src/models/OrganizationUser.ts#L34) ? it is not clear to me how can be it required (since the association with the organization may not exists)